Gruntfile.js

```diff
@@ -337,7 +364,7 @@ module.exports = function(grunt) {
 	// 4. Where we tell Grunt what to do when we type "grunt" into the terminal.
 	grunt.registerTask( 'install',             [ 'copy:hooks', 'build' ] );
 	grunt.registerTask( 'i18n',                [ 'clean', 'addtextdomain', 'makepot' ] );
-	grunt.registerTask( 'styles',              [ 'sass' ] );
+	grunt.registerTask( 'styles',              [ 'sass', 'postcss' ] );
```

Comment:
Can you switch the build task to call this task (styles) instead of sass directly? It's skipping the postcss work right now :)

Reply:
✅